web/themes/new_weather_theme/assets/sass/components/quick-forecast.scss

```diff
@@ -0,0 +1,312 @@
+@use "uswds-core" as *;
+@forward "uswds-theme";
+@forward "uswds";
+
+// Mobile is the default
+
+.wx-quick-forecast {
+    @include grid-row;
+
+    border: 1px solid color("base-lighter");
+    text-decoration: none;
+    background-color: color("white");
+
+    &:last-child {
+        border-bottom: none;
+    }
+
+    hr {
+        display: none;
+    }
+
+    time {
+        @include u-font("mono", "xs");
+        @include grid-col;
+
+        color: color("primary-dark");
+        text-align: left;
+
+        span {
+            display: block;
+        }
+        
+        span:last-child {
+            text-decoration: underline;
+            margin-top: units(0);
+        }
```

Comment:
Non-blocking: Wondering if on mobile we could shave off a little bit of the width the time block takes up so that "condition", and particularly its header, can have a bit more space. This is an enhancement so don't need to do it now.